cpu/samd5x/cpu.c

```diff
@@ -105,6 +105,41 @@ static void gclk_connect(uint8_t id, uint8_t src, uint32_t flags) {
     while (GCLK->SYNCBUSY.reg & GCLK_SYNCBUSY_GENCTRL(id)) {}
 }
 
+void sam0_gclk_enable(uint8_t id)
+{
+    /* clocks 0 & 1 are always running */
+
+    switch (id) {
+    case SAM0_GCLK_8MHZ:
+        /* 8 MHz clock used by xtimer */
+#if USE_DPLL
+        gclk_connect(SAM0_GCLK_8MHZ, GCLK_SOURCE_DPLL0, GCLK_GENCTRL_DIV(DPLL_DIV * CLOCK_CORECLOCK / 8000000));
```

Comment:
Could you split it into two lines please ?

Reply:
Ok!